fecgov/openFEC
# Upgrade webpack

## Summary (required)
I upgraded webpack. This leads to a lot of other required package upgrades and a few linting changes.

- Resolves #5325 

(Include a summary of proposed changes and connect issue below)

### Required reviewers

(Include who is required to review prior to merge. For example: One designer and two front end developer reviews are required prior to merge)
I don't know.
## Impacted areas of the application
all frontend
General components of the application that this PR will affect:

-  

## Screenshots

(Include a screenshot of the new/updated features in context (“in the wild”). If it is an interface change, include both before and after screenshots)


## How to test
run npm run develop and npm run build and make sure output is correct
(Include any information that may be helpful to the reviewer(s). This might include links to sample pages to test or any local environmental setup that is unusual such as environment variable (never credentials), API version to point to, etc)

- 

## System architecture updates (if applicable)
none
@cnlucas are you the right person to look at this one too?
